lsp/tailwindcss.lua

```diff
@@ -132,8 +132,10 @@ return {
       'app/assets/tailwind/application.css',
     }
     local fname = vim.api.nvim_buf_get_name(bufnr)
-    root_files = util.insert_package_json(root_files, 'tailwindcss', fname)
-    root_files = util.root_markers_with_field(root_files, { 'mix.lock' }, 'tailwind', fname)
-    on_dir(vim.fs.dirname(vim.fs.find(root_files, { path = fname, upward = true })[1]))
+    if vim.uv.fs_stat(fname) ~= nil then
```

Comment:
can you add a 1-line comment giving a hint
```suggestion
    -- tailwind lsp behaves poorly if the buffer is not backed by a file. https://github.com/neovim/nvim-lspconfig/pull/3875
    if vim.uv.fs_stat(fname) ~= nil then
```